Node.js/sampleresponse.js

```diff
@@ -61,13 +63,25 @@ async function msgCB(md, buf) {
 }
 
 function handleSyncPoint(buf, md , ok) {
-  
-  if (!ok) {
-    mqBoilerPlate.rollback(buf, md, poisoningMessageHandler,callbackOnRollback);
-  } else {
-    mqBoilerPlate.commit(callbackOnCommit);
-  }
-
+  //Suspending the background async get process to avoid MQRC 2500 : MQRC_HCONN_ASYNC_ACTIVE.
```

Comment:
handleSyncPoint invokes a series of promises. So it should also be either marked as async or return a promise. Which in turn affects how it is invoked in `msgCB` ie. 
```
await handleSyncPoint(buf, md, ok);
```